### PR TITLE
use a span for "kind" within the page h1

### PR DESCRIPTION
**Pull request by kevmoo**
div should not be used within an "h" tag

[Review on Reviewable](https://reviewable.io/reviews/dart-lang/dartdoc/1020)

### CODE REVIEW

**Comment by sethladd** [comment]
Nice, thanks!

LGTM

**Comment by devoncarew** [comment]
> div should not be used within an "h" tag

Good to know. Is this an html best practices, or an html spec thing?

File: lib/src/html_generator.dart [hunk 1]
```diff
@@ -466,7 +466,7 @@ abstract class TemplateData {
   String _layoutTitle(String name, String kind, bool isDeprecated) {
     if (kind.isEmpty) kind =
         '&nbsp;'; // Ugly. fixes https://github.com/dart-lang/dartdoc/issues/695
-    String str = '<div class="kind">$kind</div>';
+    String str = '<span class="kind">$kind</span>';
```
Comment (devoncarew) [inline lib/src/html_generator.dart]: Not part of your changes, but can you add braces around the if statement body above? Or move the comment one line above. It's formatting oddly.

**Comment by kevmoo** [comment]
Both.

Blocks can contain blocks and flow

Flow should only contain flow

File: lib/src/html_generator.dart (hunk 1 above)
Comment (kevmoo) [inline lib/src/html_generator.dart]: Already committed, although I agree. Do a quick pull request on this?

Comment (devoncarew) [inline lib/src/html_generator.dart]: That's what the github UI is for :)

**Pull request merged**